### DISCUSSION TITLE
Mount Duck corrupt installer

### DISCUSSION THREAD

**Discussion by pixelartes** [iterate-ch/cyberduck#16549] [Q&A]
I have just purchased Mountain Duck for mac but the unzipped file is corrupt and wont
launch. See the message I get.
![Capto_Capture 2024-11-18_12-01-46_PM2](https://github.com/user-attachments/assets/911d400c-c83f-497a-952f-057fec2b20fd)
Anyone know why this is happening?
I'm on an M3 iMac with Sonoma 14.5

**Comment by tatze96** [comment]
- Did you copy the application file into the _Applications_ folder before attempting to open it?
- Do you use any antivirus or anti-malware tools that may alter the extracted application file?

Please try if re-downloading and re-copying the application file to the _Applications_ folder resolves the error.
